addOns/graphql/src/test/java/org/zaproxy/addon/graphql/GraphQlParserUnitTest.java

```diff
@@ -46,13 +46,19 @@ public void teardown() throws Exception {
 
     @Test
     public void shouldFailIntrospectionWhenResponseIsEmpty() throws Exception {
+        // Given
         nano.addHandler(new StaticContentServerHandler("/", ""));
-        assertThrows(IOException.class, () -> new GraphQlParser(endpointUrl).introspect());
+        // When/Then
+        GraphQlParser gqp = new GraphQlParser(endpointUrl);
```

Comment:
This should be part of the given, not what's being actually tested (same for the following test).